### PR TITLE
Autofixing rubocop offenses and Problematic test for #578

### PR DESCRIPTION
**Pull request by alaxalves**
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

[//]: # (To mark checkbox write 'x' within the square brackets)

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by alaxalves** [comment]
@cesswairimu `development` branch already has #578 in it. Feel free to close it if you want to.

File: .travis.yml [hunk 1]
```diff
@@ -25,6 +25,7 @@ env:
     - TASK="CI=true TRAVIS=true rake test:integration"
     - TASK="CI=true TRAVIS=true rake test:functional"
     - TASK="CI=true TRAVIS=true rake assets:precompile"
+    - TASK="CI=true TRAVIS=true bundle exec rubocop"
```
Comment (kaustubh-nair) [inline .travis.yml]: I still dislike the idea of Rubocop blocking travis builds :/

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -41,8 +41,8 @@ def errors_for(object, message = nil)
   # see https://github.com/rails/jquery-ujs/wiki/Manual-installing-and-Rails-2
   def csrf_meta_tags
     if protect_against_forgery?
-      out = %(<meta name="csrf-param" content="%s"/>\n)
-      out << %(<meta name="csrf-token" content="%s"/>)
+      out = %(<meta name="csrf-param" content="%s"/>\n) # rubocop:disable Style/FormatStringToken
+      out << %(<meta name="csrf-token" content="%s"/>) # rubocop:disable Style/FormatStringToken
```
Comment (kaustubh-nair) [inline app/helpers/application_helper.rb]: Why not add this in `rubocop.yml` ?

File: db/schema.rb [hunk 3]
```diff
@@ -9,154 +9,153 @@
...
+  create_table "annotations", force: :cascade do |t|
+    t.integer  "map_id",          limit: 4
+    t.integer  "user_id",         limit: 4
+    t.string   "annotation_type", limit: 255
+    t.string   "text",            limit: 255
+    t.string   "style",           limit: 255
+    t.string   "coordinates",     limit: 255
+    t.datetime "created_at",                  null: false
+    t.datetime "updated_at",                  null: false
   end
```
Comment (kaustubh-nair) [inline db/schema.rb]: How did these schema changes happen without any migrations?

Comment (kaustubh-nair) [reply db/schema.rb]: Was it done by rubocop?

File: app/helpers/application_helper.rb (hunk 2 above)
Comment (alaxalves) [reply app/helpers/application_helper.rb]: Because the content of this is a crsf token.

File: db/schema.rb (hunk 3 above)
Comment (alaxalves) [reply db/schema.rb]: Yes, rubocop has formatted it. But there was no impact on database setup or anything like that.

File: .travis.yml (hunk 1 above)
Comment (alaxalves) [reply .travis.yml]: We need to have those since I have got rubocop synch'd with CodeClimate, we're getting a wider view of our PR statuses.

**Pull request merged**